modules/identity/pages/single-sign-on-with-oidc.adoc

```diff
@@ -174,7 +174,7 @@ NOTE: Activating this option means any user authorized by the IdP to access Boni
  ** `token-issuer-uri` needs to be set with the URL that will be used as issuer in the JWT tokens (`iss` claim). It can be prefixed with `+{auth-server-url}+` to only indicate the path of the issuer URI or be a full absolute URI. For example using Google's OIDC provider, the issuer is `https://accounts.google.com`. This property is specific to Bonita OIDC module and is not supported/documented by Keycloak.
  ** `user-info-url` needs to be set with the URL of your OIDC provider userInfo end point. It is only used when accessing a resource with a Bearer `Authorization` header with an access token that is opaque (not a JWT) and cannot be validated or used to retrieve the logged in user claims. It can be prefixed with `+{auth-server-url}+` to only indicate the path of the end point or be a full absolute URL. This property is specific to Bonita OIDC module and is not supported/documented by Keycloak.
  ** `jwks-url` needs to be set with the URL of your OIDC provider JSON Web Key Set endpoint. It can be prefixed with `+{auth-server-url}+` to only indicate the path of the end point or be a full absolute URL. For example with Google as OIDC provider, this URL is `https://www.googleapis.com/oauth2/v3/certs`. This property is specific to Bonita OIDC module and is not supported/documented by Keycloak.
- ** `jwt-access-token` indicates if the access token is a JWT. If false, it will be considered by Bonita OIDC module as an opaque token and its validation and retrieval of the logged in user principal name will be delegated to the userInfo endpoint (see `user-info-url`). This property is specific to Bonita OIDC module and is not supported/documented by Keycloak.
+ ** `jwt-access-token` indicates if the access token is a JWT and should be validated. If false, it will be considered by Bonita OIDC module as an opaque token, its validation won't be performed (but the ID token will, however, be validated) and the retrieval of the logged in user principal name will be delegated to the userInfo endpoint (see `user-info-url`) when accessing a resource with a Bearer `Authorization` header. Even if the access token is a JWT, this property should be set to false when the OIDC clients are not supposed/able to validate it. This property is specific to Bonita OIDC module and is not supported/documented by Keycloak.
```

Comment:
In Azure AD context, Microsoft documentation clearly explains this access token is suppose to be read / decrypted.